src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -187,7 +186,7 @@ Task ReceiveMessage(IMessageReceiver internalReceiver, BrokeredMessage message,
                 var invocations = (ConcurrentDictionary<Task, Task>) state;
                 Task toBeRemoved;
                 invocations.TryRemove(t, out toBeRemoved);
-            }, pipelineInvocations, TaskContinuationOptions.ExecuteSynchronously);
+            }, pipelineInvocations, TaskContinuationOptions.ExecuteSynchronously).Ignore();
```

Comment:
Frankly, I want to scrape it and do the [right thing](https://github.com/Particular/NServiceBus.AzureServiceBus/issues/568#issuecomment-311514606). 

No pressure @yvesgoeleven 😃